packages/console/src/pages/Connectors/index.tsx

```diff
@@ -42,30 +67,60 @@ const Connectors = () => {
               <td colSpan={2}>loading</td>
             </tr>
           )}
-          {data?.map(({ id, metadata: { name, logo }, enabled }) => (
-            <tr key={id}>
+          {!isLoading && !isSocial && (
+            <tr>
               <td>
-                <Link to={`/connectors/${id}`} className={styles.link}>
-                  <ItemPreview
-                    title={name[i18n.language] ?? name.en ?? '-'}
-                    subtitle={id}
-                    icon={
-                      logo.startsWith('http') ? (
-                        <img className={styles.logo} src={logo} />
-                      ) : (
-                        <ImagePlaceholder />
-                      )
-                    }
-                  />
-                </Link>
+                <ConnectorName
+                  connector={emailConnector}
+                  titlePlaceholder={t('connectors.type.email')}
+                />
               </td>
+              <td>{t('connectors.type.email')}</td>
               <td>
-                {enabled
-                  ? t('connectors.connector_status_enabled')
-                  : t('connectors.connector_status_disabled')}
+                {emailConnector ? (
+                  <Status status="operational">{t('connectors.connector_status_enabled')}</Status>
+                ) : (
+                  <Button title="admin_console.connectors.set_up" />
+                )}
               </td>
             </tr>
-          ))}
+          )}
+          {!isLoading && !isSocial && (
```

Comment:
most of display logic of email / sms connector are shared. consider for a better implementation?